velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -270,7 +273,9 @@ class PlanBuilder {
           connector::CommitStrategy::kNoCommit);
 
   /// Add a TableWriteMergeNode.
-  PlanBuilder& tableWriteMerge();
+  PlanBuilder& tableWriteMerge(
+      RowTypePtr outputType = nullptr,
+      std::shared_ptr<core::AggregationNode> aggregationNode = nullptr);
```

Comment:
I think we can generate outputType based on aggregationNode?

Reply:
Yeah, but the outputType from TableWriter is originally generated from aggregationNode from upper level (presto): https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp#L2139-L2159

So in lower level (velox), generating again feels redundant to me and I think we can directly reuse